spec/features/user_login_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+feature "User login" do
+  background do
+  @user = User.create(
+    fullname: "jimbobfrank",
+    email: "jimbobfrank@aol.com",
+    role:  0,
+    password: 'foobar1234',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.